requirements.txt

```diff
@@ -1,10 +1,13 @@
+azure-storage-blob==12.8.1
```

Comment:
Hopefully this won't add a massive footprint to the SDK, but I guess we'll need it eventually for collection creation too